cmake/configure/configure_cgal.cmake

```diff
@@ -17,4 +17,41 @@
 # Configuration for the CGAL library:
 #
 
+IF(NOT FEATURE_BOOST_PROCESSED)
+  MESSAGE(FATAL_ERROR "\n"
+    "Internal build system error: The configuration of "
+    "DEAL_II_WITH_CGAL depends on "
+    "DEAL_II_WITH_BOOST, but CONFIGURE_FEATURE(CGAL) "
+    "was called before CONFIGURE_FEATURE(BOOST).\n\n"
+    )
+ENDIF()
```

Comment:
@drwells This is just a safeguard ensuring that the order is maintained after configure file renaming :smile_cat:

Reply:
Yes, we should definitely still do this!